test/java/com/arpnetworking/metrics/portal/reports/impl/EmailSenderTest.java

```diff
@@ -100,17 +107,35 @@ public void testSendFailsIfExceptionThrown() throws MailException, ExecutionExce
         final RenderedReport report = TestBeanFactory.createRenderedReportBuilder().build();
         try {
             _sender.send(
-                    TestBeanFactory.createRecipient(),
+                    TestBeanFactory.createRecipientBuilder().build(),
                     ImmutableMap.of(new HtmlReportFormat.Builder().build(), report)
             ).toCompletableFuture().get();
         } catch (final ExecutionException e) {
-            if (e.getCause() instanceof MailException) {
-                throw (MailException) e.getCause();
+            final Throwable cause = e.getCause();
```

Comment:
I think this would be clearer with a JUnit rule and an `thrown.expectCause(isInstanceOf(MailException.class))` call

see https://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html#expectCause(org.hamcrest.Matcher)